--- 092wubi.schema.yaml	2025-11-02 20:13:42.925551154 +0800
+++ 2	2025-11-02 20:26:42.141647816 +0800
@@ -223,7 +223,7 @@
     - { accept: "Control+i", toggle: preedit_is_candidate, when: composing } # preedit为候选 开关
     - { when: always, accept: "Control+period" , toggle: ascii_punct } # 中英标点切换
     - { accept: space, send: Escape, when: composing } # 无候选时(空码)空格清屏
-    #- { accept: 'Shift+space', toggle: ascii_mode, when: always } # 平板上使用
+    - { accept: 'Shift+space', toggle: ascii_mode, when: always } # 平板上使用
 
 
 recognizer:
